coresdk/src/test/unit_tests/unit_test_json.cpp

```diff
@@ -117,7 +117,18 @@ TEST_CASE("json can be created and read", "[json]")
 
         REQUIRE("#00ff00ff" == color_to_string(deserialized_clr));
     }
+    SECTION("json can be created and read with different number types")
```

Comment:
This could be its own `TEST_CASE` since you're creating a new json object rather than reusing the `person` json created in the parent case.
```suggestion
TEST_CASE("json can be created and read with different number types", "[json_read_number][json_read_number_as_double]")
```
Alternatively, you could reuse the previous json object.
In this case, the test will run pretty similarly (the code in the containing TEST_CASE or SECTION will run once for each nested SECTION), so it's mostly for organisation/readability. It's worth noting in case you want to take advantage of nested SECTIONs.